networking/p2p/src/main/java/tech/pegasys/teku/networking/p2p/libp2p/LibP2PNetworkBuilder.java

```diff
@@ -156,6 +156,15 @@ protected Host createHost() {
           b.getIdentity().setFactory(() -> privKey);
           b.getTransports().add(TcpTransport::new);
           b.getSecureChannels().add(NoiseXXSecureChannel::new);
+
+          // yamux MUST take precedence during negotiation
+          if (config.isYamuxEnabled()) {
+            // TODO: setting maxBufferedConnectionWrites to 150 MiB to handle overflowing the write
```

Comment:
outstanding todo? is there a ticket?

Reply:
@StefanBratanov  It looks like the default config is already 150MiB? https://github.com/libp2p/jvm-libp2p/pull/325/files

Reply:
good point, added one https://github.com/Consensys/teku/issues/7532 will reference it in the comment

Reply:
> @StefanBratanov It looks like the default config is already 150MiB? https://github.com/libp2p/jvm-libp2p/pull/325/files

We closed that PR, agreed it's better to override it in Teku instead, since 150 MiB was excessive as a default.

Reply:
Ah my bad, didn't notice it was closed instead of merged ;)